README.md

```diff
@@ -296,6 +296,21 @@ cp contrib_loader.go.dist contrib_loader.go
 make current
 ```
 
+## Debugging
+
+If you want to use [Delve](https://github.com/derekparker/delve) for debugging you need to build gollum with some additional flags:
+
+```bash
+go build -ldflags='-s -linkmode=internal' -gcflags='-N -l'
+```
+
+With this debug build you are able to start a [Delve](https://github.com/derekparker/delve) remote debugger:
+
+```bash
+# for the gollum arguments pls use this format: ./gollum -- -c my/config.conf
+dlv --listen=:2345 --headless=true --api-version=2 --log exec ./gollum -- -c testing/configs/test_router.conf -ll 3
+```
+
```

Comment:
not really i guess - problem is the path to your config file which is different for each execution: `-c testing/configs/test_router.conf`